src/cloudant/account.py

```diff
@@ -440,16 +446,70 @@ def cors_origins(self):
         Retrieve a list of CORS origins
 
         """
-        pass
+        cors = self.cors_configuration()
+
+        return cors['origins']
 
     def update_cors_configuration(
             self,
             enable_cors=True,
             allow_credentials=True,
-            *origins):
+            origins=[],
```

Comment:
It's never safe to use a mutable value as a default argument. The code in the body of the function doesn't append or modify this particular list, so you're safe. But this function will reference the same list object every time it runs and doesn't get passed an origins argument, meaning that you're one bad refactor away from having all the users share a sort of master origins list that everyone can append to ...